### PR TITLE
base: Incall vibration options [1/3]

### PR DESCRIPTION
**Pull request by Laker87**
* allow setting vibration when call is connected
* allow setting vibration when call is disconnected
* allow setting vibration for call waiting
* this works with google and aosp dialer :)

Signed-off-by: Joey Huab <joey@evolution-x.org>

### CODE REVIEW

**Comment by gwolf2u** [comment]
we already have FOD commits
clean base?

**Comment by Laker87** [comment]
> we already have FOD commits
> clean base?

Hm. Let me check why it happened.

**Comment by gwolf2u** [comment]
just rebase
Pranav made some changes to fwb recently

**Comment by Laker87** [comment]
> just rebase
> Pranav made some changes to fwb recently

Oh, ok

**Comment by Laker87** [comment]
It seems OK now

File: core/java/android/provider/Settings.java [hunk 1]
```diff
@@ -5529,6 +5551,9 @@ public static void setShowGTalkServiceStatusForUser(ContentResolver cr, boolean
             PRIVATE_SETTINGS.add(LOCK_TO_APP_ENABLED);
             PRIVATE_SETTINGS.add(EGG_MODE);
             PRIVATE_SETTINGS.add(DISPLAY_COLOR_MODE);
+            PRIVATE_SETTINGS.add(VIBRATE_ON_CONNECT);
```
Comment (gwolf2u) [inline core/java/android/provider/Settings.java]: remove those - unsed

**Review by gwolf2u — CHANGES_REQUESTED** [review]
commits are good
only small change needed here and we can merge (tested on my guacamole and beryllium)

File: core/java/android/provider/Settings.java (hunk 1 above)
Comment (Laker87) [reply core/java/android/provider/Settings.java]: Done

**Review by gwolf2u — APPROVED** [review]

**Pull request merged**